CMakeLists.txt

```diff
@@ -8,6 +8,9 @@ find_package(ECM REQUIRED NO_MODULE)
 set(CMAKE_MODULE_PATH ${ECM_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules)
 set(CMAKE_AUTOMOC TRUE)
 
+# Akonadi Prefix: Set this to KF5 for builds with Akonadi Libs before 5.23
+set(AKO_PREFIX "KPim5")
```

Comment:
Could this be `option(AKO_PREFIX "KPim5")` instead? Then it could be change by giving the option to cmake instead of modifying the file.

Reply:
Absolutely, thank you for pointing out!